analyzers/src/SonarAnalyzer.Common/Helpers/WildcardPatternMatcher.cs

```diff
@@ -119,13 +114,9 @@ private static string ToRegexp(string wildcardPattern, string directorySeparator
                     // Single character
                     sb.Append(ch);
                 }
-
                 i++;
             }
-
-            sb.Append('$');
-
-            return sb.ToString();
+            return sb.Append('$').ToString();
```

Comment:
`WildcardPatternMatcherTest.cs`: Empty lines between `DataRows` is very hard to read

Reply:
`WildcardPatternMatcherTest.cs`: It's weird to use `.java` instead of `.cs`